Constitution.md

```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
 
 If an executive officer is dismissed, the club must follow [**Section 3: Club Changeover**](#section-3-club-changeover) in order to follow the proper procedure(s) outlined to fill the vacant position(s).
 
+## Article 2d: Committees, Councillors, and Community Advocates
+
+### Committees
+A Committee is a group of council members that forms to complete a well-defined goal. Any Executive Officer may propose a committee by outlining its purpose and the criteria of its members (if any). The Council will vote on the formation of this Committee, and upon its formation, the proposing Executive Officer will lead it.
+
+A Committee leader will have the following duties:
+1. Appoint Committee members from Council.
+2. If applicable, announce the formation of the Committee to Members of the Association so that they may apply for it.
+3. Outline the necessary tasks for the Committee to complete its goal.
+4. Assign tasks to Committee members.
+5. Call Committee meetings.
+6. Report on Committee activites and progress at Council meetings.
+7. Appoint a proxy Committee leader in the event that they become unavailable to lead.
+
+If a Committee leader deems the Committee's goal to be fulfilled, they may propose its dissolution to be voted on by the Council.
+
+### Councillors
+Councillors are non-voting members of council. Members of CSSA may apply for a Councillor position through an application that is publicly available throughout the year. Councillor applicants may indicate their interest to join an active committee. Once appointed by the Council, a Councillor may assume a general purpose volunteer role, or be appointed to a committee that they express interest in.
+
+### Community Advocates
+Applicants will have the option to self-nominate for a Community Advocate position on their Councillor application. However, self-nominations for Community Advocates will close [WITHIN SOME TIME LIMIT], after which the option will be removed from application form.
```

Comment:
question: is this a separate form from councillors?

ps. don't forget to fill in the time limit :P

Reply:
Can we arbitrarily define that these should be done before the start of community assemblies or is that too vague

Reply:
1. I was thinking the same form but with a question for self-nom, then that question gets removed after the time limit.
2. "before community assemblies" is a touch vague for my liking, how about tentatively end of september? I'll put that in for now and we'll see what thoughts people have